account-kit/java/signer-sdk/src/main/java/com/alchemy/aa/core/TekManager.java

```diff
@@ -72,7 +60,7 @@ private KeysetHandle getKeysetHandle() {
     /**
      * Get hpke public key from stored keyset handle.
      *
-     * @return hpke public key or null if createTEK is never called.
+     * @return hpke public key or null if createTek is never called.
```

Comment:
Why do we allow returning null to be a case? Why not call `createTek` if it hasn't been called yet?

Reply:
amend comments